tutorials/04_physics_engines.md

```diff
@@ -1,3 +1,3 @@
 \page physicsengine Use different physics engines
 
-See [Physics engines tutorial](https://gazebosim.org/api/gazebo/3.8/physics.html) in Gazebo.
+See [Physics engines tutorial](https://gazebosim.org/api/sim/9/physics.html) in Gazebo.
```

Comment:
We should be able to use `\ref` commands for these, but I won't block this PR.

Reply:
I see, let me change that quickly

Reply:
Verified offline that this is not possible since gz-sim is downstream from gz-physics.